snap/local/bin/source-env-file.sh

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash -e
 
-SERVICE="app-service-configurable"
-ENV_FILE="$SNAP_DATA/config/res/$SERVICE.env"
+# convert cmdline to string array
+ARGV=($@)
+
+# grab binary path
+BINPATH="${ARGV[0]}"
+
+# binary name == service name/key
+SERVICE=$(basename "$BINPATH")
+ENV_FILE="$SNAP_DATA/config/$SERVICE/overrides.env"
```

Comment:
The given path is incorrect. See https://github.com/canonical/edgex-snap-hooks/blob/544c6823a1b007cf18e22cfa85d1ba12b0ffa2ba/options/env.go#L81